crates/egui/src/containers/window.rs

```diff
@@ -514,10 +514,13 @@ impl<'open> Window<'open> {
             if let Some(title_bar) = title_bar {
                 if on_top && area_content_ui.visuals().window_highlight_topmost {
                     let rect = Rect::from_min_size(
-                        outer_rect.min,
```

Comment:
I think you can keep the original code to create the `Rect` and add `.shrink(frame_stroke.width)` at the end. Should make the intent more clear.